.github/workflows/go.yml

```diff
@@ -5,9 +5,8 @@ name: Go
 
 on:
   push:
-    branches-ignore: [ "main" ]
-  pull_request:
-    branches-ignore: [ "main" ]
+    tags: ['v[0-9]+.[0-9]+.[0-9]+']
```

Comment:
the tags should follow the naming convention ionos-exporter-[0-9]+.[0-9]+.[0-9]+
see for example https://github.com/hpi-schul-cloud/infra-tools/blob/master/.github/workflows/bapontag.yaml